pkg/policyGovernance/security/imageScanning/ImageScanService.go

```diff
@@ -249,13 +257,40 @@ func (impl ImageScanServiceImpl) fetchImageExecutionHistoryMapByIds(historyIds [
 	}
 	return mapOfExecutionHistoryIdVsExecutionTime, nil
 }
+func (impl ImageScanServiceImpl) fetchResourceScanningDataForEnv(appId, envId int) (int, error) {
```

Comment:
func name and implementation is different, we can change the func name

Reply:
DONE